ovh/resource_ovh_domain_zone_record.go

```diff
@@ -91,6 +107,9 @@ func resourceOvhDomainZoneRecordRead(d *schema.ResourceData, meta interface{}) e
 	provider := meta.(*Config)
 
 	record := OvhDomainZoneRecord{}
+	if logging.IsDebugOrHigher() {
```

Comment:
we rarely see this kind of perf enhancer in the terraform code base. maybe we could get rid of it

Reply:
I put it for Debug in fact and forget to remove. The idea behind was to understand how others providers when in debug managed to send API query they were doing which is in fact nice to understand what's going on. I removed it